requirements.txt

```diff
@@ -1,7 +1,5 @@
 click==7.1.2
 dask[dataframe]
 Flask==1.1.2
-modelmeta==0.2.0
-psycopg2==2.8.6
-SQLAlchemy==1.3.23
+netCDF4==1.5.6
```

Comment:
Where is this used?

Reply:
I think its a part of `xarray` or `dask`. Tests were failing without specifically adding it in.

Reply:
oh weird, I'm pretty sure xarray uses it but I thought it was a dependency